gen/suites/transfer/main.go

```diff
@@ -155,4 +157,67 @@ func main() {
 			Func: failTransferUnknownSenderUnknownReceiver,
 		},
 	)
+
+	g.MessageVectorGroup("system_receiver",
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-system-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.SystemActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-init-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.InitActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-reward-actor",
+				Version: "v1",
+			},
+			Func:  transferToSystemActor(builtin.RewardActorAddr),
+			Mode:  ModeLenientAssertions,
+			Hints: []string{schema.HintIncorrect},
+		},
```

Comment:
Gotta sort this out as we should be asserting the actual balance taking into account mining rewards.